src/main/java/roomescape/service/JWTProvider.java

```diff
@@ -0,0 +1,36 @@
+package roomescape.service;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.stereotype.Service;
+import roomescape.exception.AuthorizationException;
+
+import java.util.Date;
+
+@Service
+public class JWTProvider {
+    private static final String secretKey = "learning-test-spring";
+    private static final long validityInMilliseconds = 3600000;
```

Comment:
정적 상수는 대문자로 선언해주는 것이 일반적입니다!

```suggestion
    private static final String SECRET_KEY = "learning-test-spring";
    private static final long VALIDITY_IN_MILLISECONDS = 3600000;
```